### PR TITLE
Pyinstaller for windows

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

**Comment by SLdragon** [comment]
> https://pyinstaller.readthedocs.io/en/stable/runtime-information.html#ld-library-path-libpath-considerations still not addressed like in docker compose?

This is used by Linux and I think when we can add it when we develop pyisntaller for Linux. Because it would change search path env, not sure whether it would bring other issues.

File: vsts_ci/standalone-binaries/continuous-build-standalone-binaries-win32.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      architecture: x86
+
+  - powershell: |
+      $env:Path += ";C:\Program Files (x86)\Windows Kits\10\Redist\ucrt\DLLs\x86"
+      $VERSION_TAG = git describe --tags
+      Write-Host "##vso[task.setvariable variable=VERSION_TAG]$VERSION_TAG"
+      python -m venv ./venv
+      .\venv\Scripts\activate
+      python -m pip install --upgrade pip
+      if ("$(BUILD.SOURCEBRANCH)" -match "^refs/tags/v?[0-9]+\.[0-9]+\.[0-9]+$") { ((Get-Content -path "$(BUILD.REPOSITORY.LOCALPATH)\iotedgehubdev\__init__.py" -Raw) -replace "__AIkey__ = '.*'","__AIkey__ = '$(AI_KEY)'") | Set-Content -Path "$(BUILD.REPOSITORY.LOCALPATH)\iotedgehubdev\__init__.py" }
```
Comment (adashen) [inline vsts_ci/standalone-binaries/continuous-build-standalone-binaries-win32.yml]: rc build will not has package?

File: vsts_ci/azure-pipelines.yml [hunk 2]
```diff
@@ -23,6 +23,13 @@ jobs:
     steps:
       - template: policy/continuous-legal-status-policy-check.yml
 
+  - job: PublishStandaloneBinariesForWin32
```
Comment (adashen) [inline vsts_ci/azure-pipelines.yml]: will this job be triggered during for PR?

Comment (SLdragon) [reply vsts_ci/azure-pipelines.yml]: Just when it has tags

**Pull request merged**